views/js/test/testMetaData/test.js

```diff
@@ -0,0 +1,142 @@
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+require([
+    'lodash',
+    'jquery',
+    'taoQtiTest/testRunner/testMetaData'
+], function (_, $, TestMetaData) {
+    'use strict';
+
+    QUnit.test("Constructor", function () {
+        var testServiceCallId = "http://sample/first.rdf#i14435993288775133.item-1.0";
+        QUnit.assert.throws(
```

Comment:
Please use the following format, in order to facilitate the coming migration to QUnit 2

```js
QUnit.test("Constructor", function (assert) {
    var testServiceCallId = "http://sample/first.rdf#i14435993288775133.item-1.0";
    assert.throws(function(){
    //...
```